deployments/container/Dockerfile.ubi8

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We use the ubi8-minimal image as a reference image in removing unneeded dependencies.
+FROM redhat/ubi8-minimal:latest AS minimal
+
+RUN rpm -qa --queryformat='^%{NAME}-\[0-9\].*\.%{ARCH}$\n' | sort -u > /tmp/minimal-list.txt
```

Comment:
Can we merge these two ?

Reply:
I would prefer not to as I find two different commands more readable. I understand that we're generating two separate layers here, but this is in an intermediate image that doesn't affect the final image that we produce.

If we're really concerned about layers in the images, we could squash the image as a final step in our pipelines.